domain_entity.module

```diff
@@ -69,7 +69,7 @@ function domain_entity_field_default_domains(FieldableEntityInterface $entity, F
 function domain_entity_form_field_storage_config_edit_form_alter(&$form, FormStateInterface $form_state) {
   /** @var \Drupal\field\Entity\FieldStorageConfig $field_storage */
   $field_storage = $form_state->getFormObject()->getEntity();
-  if ($field_storage->getName() === DomainEntityMapper::FIELD_NAME) {
+  if ($field_storage->getName() === DOMAIN_ACCESS_FIELD) {
```

Comment:
I'm against of this change because of https://github.com/agentrickard/domain/issues/337